src/screens/TxDetails.js

```diff
@@ -106,7 +106,7 @@ export class TxDetailsView extends React.PureComponent {
           networkKey={this.props.recipient.networkKey || ''}
         />
         <Button
-          buttonStyles={{ backgroundColor: colors.bg_positive, height: 60 }}
+          buttonStyles={{ height: 60 }}
           title="Sign Transaction"
           textStyles={{ color: colors.card_text }}
```

Comment:
found another nit!
the "Sign transaction" button is written black on blue whereas all others are white on blue.  Can we make sure to have the same styling?